android/app/src/main/kotlin/com/chicagoroboto/features/main/MainActivity.kt

```diff
@@ -34,40 +28,26 @@ class MainActivity : AppCompatActivity(), SessionNavigator, SpeakerNavigator, Na
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setTitle(R.string.event_name)
+
         component = getAppComponent().mainComponent(MainModule(this, this))
 
         setContentView(R.layout.activity_main)
-        if (SDK_INT >= LOLLIPOP) {
-            findViewById(android.R.id.content).systemUiVisibility = SYSTEM_UI_FLAG_LAYOUT_STABLE or SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN
-            val doesNotConsume: (View, WindowInsets) -> WindowInsets = { v, insets ->
-                v.onApplyWindowInsets(insets)
-                insets
-            }
-            toolbar.setOnApplyWindowInsetsListener(doesNotConsume)
-            nav_view.getHeaderView(0).setOnApplyWindowInsetsListener(doesNotConsume)
-        }
 
         setSupportActionBar(toolbar)
 
-        val toggle = ActionBarDrawerToggle(this, drawer_layout, toolbar, R.string.open_drawer, R.string.close_drawer)
-        drawer_layout.addDrawerListener(toggle)
-        toggle.syncState()
+        supportActionBar?.let {
+            it.setHomeAsUpIndicator(R.drawable.ic_menu)
+            it.setDisplayHomeAsUpEnabled(true)
+        }
+
+        setTitle(R.string.event_name)
 
         showView(R.id.action_schedule)
 
         nav_view.setNavigationItemSelectedListener(this)
         nav_view.setCheckedItem(R.id.action_schedule)
     }
 
-    override fun onBackPressed() {
```

Comment:
Pressing back with the nav drawer open should close it.  This removes that but it's not handled elsewhere.